test/jenkins/criu-iter.sh

```diff
@@ -5,4 +5,6 @@ set -e
 source `dirname $0`/criu-lib.sh
 prep
 mount_tmpfs_to_dump
-./test/zdtm.py run --all --keep-going --report report --parallel 4 --iter 3 -x 'maps04' || fail
+
+ITER_EXCLUDE="-x netns_lock -x maps04"
```

Comment:
pls add a comment that explains why we skip netns_lock.

Reply:
I went with @rst0git loop method